etna/core/models/settings.py

```diff
@@ -12,7 +13,35 @@
 
 @register_setting(icon="list-ul")
 class SiteSettings(BaseSetting, ClusterableModel):
+    beta_banner_standfirst = models.CharField(
+        max_length=200,
+        default="What does the 'beta' label mean?",
+        verbose_name="standfirst",
+    )
+    beta_banner_link = models.URLField(blank=True, verbose_name="link to")
+    beta_banner_link_text = models.CharField(
+        max_length=200,
+        default="Find out more ?",
+        verbose_name="linktext",
```

Comment:
`verbose_name` is what ultimately becomes the field label in the form. While `standfirst` is a recognised term, I'm not sure the same can be said for `"linktext"`, so perhaps we should change to:
 ```suggestion
        verbose_name="link text",
```